torch/_subclasses/fake_tensor.py

```diff
@@ -1313,6 +1314,49 @@ def tolist(self):
 # memory should not significantly increase.
 
 
+# Note - [On fake tensor policy and fresh fake modes for backends]
+#
+# FakeTensorMode does memoization - this memoization is generally fine, but there are some cases
+# where we want to disable memoization in favor of producing fake tensors anew. In dynamo, the
```

Comment:
but you aren't "disabling" memoization, you're making a new fake tensor mode, no?